Core/HLE/ReplaceTables.cpp

```diff
@@ -231,6 +248,9 @@ static int Replace_memcpy_jak() {
 		}
 	}
 
+	if (sliced) {
+		return 5 + bytes * -8 + 2;
```

Comment:
This needs a comment that a negative value is used to signal Comp_ReplacementFunc that the function call is to be repeated.